spinta/components.py

```diff
@@ -878,6 +878,7 @@ class Config:
     data_path: pathlib.Path
     AccessLog: Type[AccessLog]
     exporters: Dict[str, Format]
+    check_filename: List[str] = None
```

Comment:
```suggestion
    check_filename: Optional[bool] = None
```

This should be a boolean flag.